src/tx/applyTx/utils.js

```diff
@@ -88,12 +100,15 @@ const removeInputs = ({ unspent, balances, owners }, tx) => {
     const outpointId = input.prevout.hex();
     const { address, value, color } = unspent[outpointId];
 
-    if (Output.isNFT(color)) {
+    if (isNFT(color)) {
       const index = balances[color][address].indexOf(value);
       balances[color][address].splice(index, 1);
-      delete owners[color][value];
+      delete owners[color][BigInt(value).toString()];
```

Comment:
subtle thing here. Using `delete owners[color][value];` would just silently do nothing. So good we have unit tests 🎉